core/src/test/java/io/confluent/rest/HstsIntegrationTest.java

```diff
@@ -60,6 +62,18 @@ public class HstsIntegrationTest {
   private Properties props;
   private File clientKeystore;
 
+  HttpClient httpClient(SslContextFactory.Client sslContextFactory) {
```

Comment:
is there a reason to duplicate `httpClient()` in [ErrorHandlerIntegrationTest.java](https://github.com/confluentinc/rest-utils/pull/552/files/3f66161d8627842ec24f92c5ca181b3f1c43e602..ae462b74e51e58aecd263b8432f11bb1da40b819#diff-9de9ec813d1515d128281c576ab6dde4e5fc3f92e14757325294b2123cb558e4)? you an put the method as static func in utils class ex - `core/src/test/java/io/confluent/rest/TestUtils.java`?